store/ssmstore.go

```diff
@@ -40,15 +42,17 @@ type SSMStore struct {
 }
 
 // NewSSMStore creates a new SSMStore
-func NewSSMStore(numRetries int) (*SSMStore, error) {
+func NewSSMStore(numRetries int, minThrottleDelay time.Duration) (*SSMStore, error) {
```

Comment:
This is part of the public API and cannot be changed without a version bump. Please create a new function `NewSSMStoreWithMinThrottleDelay`